.github/workflows/pull.yml

```diff
@@ -191,15 +173,13 @@ jobs:
         uses: actions/upload-artifact@v4
         if: failure()
         with:
-          name: playwright-report
```

Comment:
this would have caused a clash in the very unlikely event that we had failures in both at once .. hasn't happened, but good practice, esp as I have unleashed all E2Es at once 👿